engine/graphics/src/null/glsl_uniform_parser.cpp

```diff
@@ -129,7 +157,16 @@ namespace dmGraphics
                     {
                         // Check name
                         NextWord(&word_start, &word_end, &size);
-                        cb(word_start, size-1, type, userdata);
+
+                        // Check if it's an array
+                        uint32_t uniform_size = 1;
+                        if (CheckUniformSize(word_start, &uniform_size))
+                        {
+                            const char* array_begin = FindChar(word_start, '[');
+                            size = array_begin - word_start + 1;
+                        }
+
+                        cb(word_start, size-1, type, uniform_size, userdata);
```

Comment:
Added support for graphics_null to get the uniform array size, just like the runtime version, in order to make the tests work.